Sources/NukeUI/VideoPlayerView.swift

```diff
@@ -18,6 +18,9 @@ public final class VideoPlayerView: _PlatformBaseView {
         }
     }
 
+    /// `true` by default. If disabled, the video will resize with the frame without animations
+    public var animatesFrameChanges = true
```

Comment:
I wasn't sure if this was really a bug or a feature enhancement, so made this configurable and defaulted to the existing behavior to avoid too much disruption for users who may expect the existing resizing animation.